infrastructure/shifty.bicep

```diff
@@ -0,0 +1,41 @@
+param location string = resourceGroup().location
+
+param environment string
+
+param targetBranch string
+
+param organizationPrefix string
+param applicationPrefix string
+
+param sharedResourceGroupName string
+
+resource staticwebapp 'Microsoft.Web/staticSites@2022-03-01' = {
+  name: 'stapp-${organizationPrefix}-${applicationPrefix}-${environment}'
+  location: location
+  sku: {
+    name: 'Free'
+    tier: 'Free'
+  }
+  properties: {
+    allowConfigFileUpdates: false
+    repositoryUrl: 'https://github.com/AnalogIO/shifty-webapp'
+    branch: targetBranch
```

Comment:
Honestly not quite sure about this "branch" keyword it seems like it's required, but I don't think it matters? the artifact we publish for deployment is built from the branch the workflow runs on, so I don't know really - I jsut went for this "targetBranch" because it felt more correct than just leaving it as "develop"